addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/DeviceHandler.java

```diff
@@ -367,6 +380,19 @@ private void updateThingChannelState(Thing thing, String channelId, State state)
     }
 
     /**
+     * Updates thing properties.
+     * 
+     * @param thing Thing which property to be updated.
+     * @param propertyId ID of the property to be updated.
+     * @param propertyValue Value to be set.
+     */
+    private void updateThingProperty(Thing thing, @NonNull String propertyId, String propertyValue) {
+        if (thing.getProperties().containsKey(propertyId)) {
```

Comment:
See earlier remark, if this check is really needed is there some re-use possible?

Reply:
I don't think this check is necessary. My intention was not to update a property which doesn't exist. But that is nonsense.